content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -82,27 +82,184 @@ Costing method
+             So, `$144 / 12 = $12`.
+
+         - When delivering ten products, the average **Unit Cost** is used to calculate the
+           **Inventory Value**, regardless of the purchase price of the product. Therefore,
+           inventory value is `$144 + (-10 * $12)` = `$144 - $120` = `$24`
+
+         - Receive two products for $6 each:
+
+           - **Inventory Value**: `$24 + (2 * $6)` = `$24 + $12` = `$36`
+           - **Unit Cost**: `$36 / 4 = $9`
+
+      .. note::
+         When choosing :guilabel:`Average Cost (AVCO)` as the :guilabel:`Costing Method`, changing
+         the numerical value in the *Cost* field for products in the respective product
+         category creates a new record in the *Inventory Valuation* report to adjust the value of
+         the product. The *Cost* amount is then automatically updated, based on the
+         average purchase price of both the inventory on-hand and the costs accumulated from validated
+         purchase orders.
+
+   .. tab:: First In First Out (FIFO)
+
+      Tracks the costs of incoming and outgoing items in real-time, and uses the real price of the
+      products to change the valuation. The oldest purchase price is used as the cost for the next
+      good sold, until an entire lot of that product is sold. When the next inventory lot moves up in
+      the queue, an updated product cost is used based on the valuation of that specific lot.
+
+      This method is arguably the most accurate inventory valuation method for a variety of reasons,
+      but it is highly sensitive to input data and human error.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $0
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $12
+           - 12
+           - 4 * $16
+           - $144
+         * - Deliver 10 products
+           - $16
+           - 2
+           - | -8 * $10
+             | -2 * $16
+           - $32
+         * - Receive 2 products for $6/unit
+           - $11
+           - 4
+           - 2 * $6
+           - $44
+
+      .. spoiler:: How are Unit Cost and Inventory Value calculated at each step?
+
+         - When receiving four products for $16 each (process is similar to the :guilabel:`Average Cost (AVCO)`
+           method in this step):
+
+           - **Inventory Value** is calculated by adding the previous inventory value to the
+             incoming value. So, with `$80 + (4 * $16)` = `$80 + $64` = `$144`.
+           - **Unit Cost** is calculated by dividing the inventory value by the quantity on-hand.
+             So, `$144 / 12 = $12`.
```

Comment:
Consider removing "process is similar to the Average Cost (AVCO)" just to keep these examples more skimmable.
```suggestion
      - When receiving four products for $16 each:

        - Inventory value is calculated by adding the previous inventory value to the incoming
          value: :math:`$80 + (4 * $16) = $144`.
        - Unit cost is calculated by dividing the inventory value by the quantity on-hand:
          :math:`$144 / 12 = $12`.
```